kubernetes/configs/dev/dtd_catalog_exporter.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-DTD_CATALOG_STORAGE_BUCKET="interop-dtd-share-dev"
+DTD_CATALOG_STORAGE_BUCKET="interop-public-catalog-dev"
```

Comment:
is this a public bucket?

Reply:
Nope, private bucket with the landing page CDN serving its content

Reply:
🔝